edge-functions/ua-based-app/components/Home.tsx

```diff
@@ -0,0 +1,107 @@
+import Head from 'next/head'
+import { Layout, Text, Page, Code, Snippet } from '@vercel/examples-ui'
+import { useRouter } from 'next/router'
+
+function Home() {
+  const { route } = useRouter()
+  const viewport = route.replace('/_viewport/', '')
+
+  return (
+    <Page>
+      <Head>
+        <title>Load pages based on users UA - Vercel Example</title>
+        <meta
+          name="description"
+          content="Vercel example on how to load pages based on users UA"
+        />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <section className="flex flex-col gap-6">
+        <Text variant="h1">Load a pages based on users UA</Text>
+        <Text>
+          Sometimes the desktop version of our application differs a lot from
+          our mobile version, because the UI is different or because we load
+          different scripts, styles, etc. We want to decide which page to load
+          based on users UA without loading unnecesary assets for the current
+          viewport.
+        </Text>
+      </section>
+
+      <section className="flex flex-col gap-6 mt-12">
+        <Text variant="h2">Folder structure</Text>
+        <Text>
+          We will rewrite our user to different pages based on its UA so we have
```

Comment:
Maybe adding a link to a UA description could help educate people who are unfamiliar with the concept.